src/ComboBox/utils.jsx

```diff
@@ -64,6 +65,43 @@ function buildListBoxOptions( options = [], prefix )
     } );
 }
 
+/**
+ * ## getScrollParent
+ * Gets the first scrollable DOM ancestor of the element
+ *
+ * @param   {HTMLElement}  el   element to find scroll parent of
+ *
+ * @return  {HTMLElement}
+ *
+ */
+function getScrollParent( element )
+{
+    let style = getComputedStyle( element );
+
+    if ( style.position === 'fixed' )
+    {
+        return document.body;
```

Comment:
We have two instances of `return document.body` in this function. Would be good to restructure it so that there‘s only one.